backend/packages/Upgrade/src/api/controllers/validators/SegmentPaginatedParamsValidator.ts

```diff
@@ -0,0 +1,64 @@
+import { Type } from 'class-transformer';
+import { IsNotEmpty, IsNumber, IsOptional, ValidateNested, IsEnum, IsString } from 'class-validator';
+import { SORT_AS_DIRECTION } from 'upgrade_types';
+
+// TODO: Move to upgrade types
```

Comment:
does this TODO need resolved?

Reply:
Probably eventually. I copied that over from the feature flag file; moving those interfaces broke stuff, but it's something that both the frontend and the backend use.